queries.sql

```diff
@@ -1,3 +1,9 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
```

Comment:
- [ ] Let us kindly apply the same here
![Screenshot from 2023-05-02 13-15-18](https://user-images.githubusercontent.com/46388113/235675617-5e748a67-07a8-4034-a04c-8e68d46b2cf6.png)